dist/constants.js

```diff
@@ -0,0 +1,7 @@
+import { canvas } from "./rendering.js";
+
+// TODO: move more stuff here
+export var hitbox = {
```

Comment:
Better to have these kind of objects, rather than arrays everywhere

Reply:
well it looks like i've got work to do. Thanks, it surely will look much better after it's properly seperated